### DISCUSSION TITLE
Project Advancement

### DISCUSSION THREAD

**Discussion by Andrea-Yzeiri** [McGill-MMA-EnterpriseAnalytics/Dazed-Confusion-Matrix#4] [Ideas]
Based on the meeting on March 15th, 2021:

- Predictions on when a crime will occur 
- User input going into the model 
- Priority with the 911 calls. Merging with 911 call dataset to determine how many calls lead to no crime
- Replace the neighborhoods with average income and look into correlation

**Comment by tipsytc** [comment]
I took another look for 911 calls that led to no crime, only aspects we can look into are priority, time and address, unless we link to more demographic data. Could make a simple visualization on percentage of no-crime calls based on neighborhood/district tho.

**Reply by Andrea-Yzeiri** [reply]
Then what if we find a crime statistic that outlines how much crime there is and work backwards to create synthetic inputs with the remaining 911 calls? (ex. Baltimore has 73% of crime on average, so our dataset in a year would contain that 73% of arrests, and we can make an inference into generating other 27% being no crime calls).

**Reply by Andrea-Yzeiri** [reply]
For example, maybe something from Numbeo, which aggregates data from around the world in a crime index. https://www.numbeo.com/crime/in/Baltimore

**Reply by tipsytc** [reply]
I've been trying a couple of things. There doesn't seem to be a clear and valid point we can make with the no crime 911 calls analysis. The dataset is not as clean as the crime one as well, where multiple calls could be reporting the same incident but lead to no crime...
